docs/strategies/elixir.md

```diff
@@ -0,0 +1,136 @@
+# Elixir Analysis
+
+When developing in Elixir, [Mix](https://hexdocs.pm/mix/Mix.html) and [Hex](https://hex.pm/) are most commonly used to manage dependencies. 
+
+| Strategy | Direct Deps        | Deep Deps          | Edges              |
+| -------- | ------------------ | ------------------ | ------------------ |
+| mix deps | :white_check_mark: | :white_check_mark: | :white_check_mark: |
+
+## Project Discovery
+
+In order to find elixir projects, we look for `mix.exs` file which specify the root of a elixir project. We skip `deps` and `_build` directories. 
+
+## Analysis
+
+1. Run `mix deps.tree --format plain` and generate output similar to:
+```
+├─ one─1.0.0 (https://github.com/dep/one)
+│  ├─ two─2.0.0 (hex package)
+│  │  └─ three─3.0.0 (hex package)
+│  └─ four─4.0.0 (https://github.com/dep/four)
+└─ five─5.0.0 (hex package)
+```
+2. Run `mix deps --all` and generate output similar to:
```

Comment:
If we've changed our commands to handle prod vs. dev differently, we should sync these docs with those changes.  if not, then these are OK.

Reply:
No we need to explicitly say `--all`. If not done so, environment variable `MIX_ENV` can influence results returned. This looks right to me.

Reply:
It looks like the first command changed to include prod, right?  you added `--only prod`

Reply:
That's correct. I want to capture all resolved dependencies with `mix deps --all`. 

The code only reports dependencies from `mix deps.tree --only prod` but uses resolved versions captured from `mix deps --all`.

Reply:
I might not be as clear as I think I'm being.  See [this comment](https://github.com/fossas/spectrometer/pull/287/files/99cea438c20e58879ceb761d421f18f47b4e658b#r674192081) for what I was expecting.

Reply:
I just noticed that this is done in the latest commit, and we're commenting on an old commit.